packages/internet-header/src/components/post-internet-header/post-internet-header.scss

```diff
@@ -72,7 +72,8 @@
   display: flex;
 
   & > .main-navigation-controls {
-    flex: 0 0 auto;
+    flex: 0 1 auto;
+    overflow: hidden;
```

Comment:
Not sure if this is necessary, slotted content does not seem to be affected by it anyways (tried to apply `position: absolute;top: 200px` to the `<p>` and it was still visible).

```suggestion
```